kernel/mm.h

```diff
@@ -0,0 +1,24 @@
+#ifndef MM_H
+#define MM_H
+
+#include "kernel.h"
+
+#define FRAME 0x1000 /* 4k */
+#define DIR_SIZE 1<<10 /* 0x400 or 1024 entries */
+
+uint32_t pdir[DIR_SIZE] __attribute__((aligned(FRAME)));
+uint32_t pt[DIR_SIZE]   __attribute__((aligned(FRAME)));
+
+#define turn_on_paging                                  \
```

Comment:
maybe add "()" at the end for completness

Reply:
well this macro take no parameters so I don't want to create confusion...